### ISSUE TITLE
Users have no way to create new wallets with a given path using the dialog

### ISSUE THREAD

**Issue by jidanni** [spesmilo/electrum#3561]
Users cannot CREATE a wallet with a given path with this dialog (Debian).
They can only open PREVIOUSLY created wallets with it.
![bx](https://user-images.githubusercontent.com/1230959/34183032-a7d041a6-e553-11e7-8992-14935737d26f.jpg)
Users cannot create a wallet at a random /a/b/c/d/e.dat if that file does not exist yet. Chicken and egg problem.

**Comment by jidanni** [comment]
So to create a new wallet at a random location, the user should never enter the second (choose location) dialog. So maybe say "Enter the full path, or choose another already existing file."

**Comment by AbdussamadA** [comment]
If that is changed to a save file dialog box people will create their wallets in unusual locations and then complain that they can't find  the wallet file. 

If you want to customize the file name just enter a different file name in the wallet field and don't click on choose.

If you want to customize the path then use -w command line option or simply create the wallet file normally and use file > save copy to save a copy where you want it.

**Comment by jidanni** [comment]
Yes, but they think that if they want to customize the whole path, not just the filename, they have no choice but enter the file selector. When in fact they can just paste in the whole path they want to create.

**Comment by jidanni** [comment]
Also note once the user creates a wallet with a path. further wallets created with File > New/Restore will be made with that path's directory. I.e., there is a side effect to entering a full path.
